app/javascript/dashboard/mixins/time.js

```diff
@@ -12,5 +12,29 @@ export default {
       const unixTime = fromUnixTime(time);
       return formatDistanceToNow(unixTime, { addSuffix: true });
     },
+    shortTimestamp(time) {
+      const timestamp = time;
+      const convertToShortTime = timestamp
+        .replace('about', '')
```

Comment:
These can be collapsed in lesser replaces with regex

replace(/about|another|word|here/g, 'world');

Reply:
Shall we use this helper and take the first letter after the space? https://date-fns.org/v2.29.3/docs/formatDistanceToNowStrict

Reply:
@pranavrajs Here, we want to replace `" year ago"` with `"y"`, is it possible with [formatDistanceToNowStrict](https://date-fns.org/v2.29.3/docs/formatDistanceToNowStrict)? Please let me know if I'm wrong.

Reply:
> Shall we use this helper and take the first letter after the space? https://date-fns.org/v2.29.3/docs/formatDistanceToNowStrict

@pranavrajs This will not work for `month` and `minute`